src/bootstrap/month-sheet-bootstrapper.js

```diff
@@ -0,0 +1,150 @@
+class MonthSheetBootstrapper extends BaseSheetBootstrapper {
+  createSheet() {
+    const monthIndex = MONTHS.indexOf(this.sheetName);
+
+    const rule1 = SpreadsheetApp.newDataValidation()
+      .requireValueInList(['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'])
+      .build();
+    this.sheet.getRange('C1').setDataValidation(rule1);
+    this.sheet.getRange('C1').setValue('Jan');
+
+    const numRows = 200;
+    const letterExpensesAmount = colToLetter(MonthSheet.colExpAmount);
+    const letterIncomeAmount = colToLetter(MonthSheet.colIncAmount);
+    const letterTransAmount = colToLetter(TransactionsSheet.columnAmmount);
+    const letterTransDate = colToLetter(TransactionsSheet.columnDate);
+    const letterTransIsInvestment = colToLetter(TransactionsSheet.columnIsInvestment);
+    this.summaryTable = new DataTable(this.sheet, 1, 2, 'Summary', ['Category', 'Total'], 3)
+      .withoutHeaders()
+      .initialize([
+        ['Income', `=SUM(${letterIncomeAmount}24:${letterIncomeAmount})`],
+        ['Expenses', `=SUM(${letterExpensesAmount}24:${letterExpensesAmount})`],
+        [
+          'Investment',
+          `=sumifs('All-Transactions'!${letterTransAmount}:${letterTransAmount}, 'All-Transactions'!${letterTransIsInvestment}:${letterTransIsInvestment}, "Yes", arrayformula(Month('All-Transactions'!${letterTransDate}:${letterTransDate})),month($C$1&1) )`,
+        ],
+      ]);
+
+    const letterColDate = colToLetter(TransactionsSheet.columnDate);
+    const letterColSource = colToLetter(TransactionsSheet.columnSource);
+    const letterColIncome = colToLetter(TransactionsSheet.columnIsIncome);
+    const letterColInvestment = colToLetter(TransactionsSheet.columnIsInvestment);
+
+    this.incomeTable = new DataTable(
+      this.sheet,
+      21,
+      9,
+      'Income',
+      ['Date', 'Amount', 'Description', 'Category', 'Classificator', 'Source'],
+      numRows
+    )
+      .withDataValidationRules([new ColumnValidationRule(5, this.categoryValidationRule)])
+      .initialize([
+        [
+          `=FILTER('All-Transactions'!${letterColDate}:${letterColSource}, Month('All-Transactions'!${letterColDate}:${letterColDate}
+          ) = month($C$1&2), 'All-Transactions'!${letterColIncome}:${letterColIncome} = "Yes", 'All-Transactions'!${letterColInvestment}:${letterColInvestment} = "No")`,
```

Comment:
Income is showing as negative values. We should convert them to positive.
(And the same applies to the summary table)